File: BY-PYTHON/primenumber.py
##  create  a python code to chek the number is prime or not 
num = int(input("Enter  a number :"))
flag = 0

if num  > 2 :
    print("NUmber is NOt  Prime ")
    
else :
    for i in range(2,num):
        if num % i == 0:
            flag = 1
            break
        
    if flag == 1:
        print("Number is not prime")
        
    else:
        print("Number is Prime")

File: JAVA/primenumber.java
import java.util.Scanner;

public class primenumber {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter  a number : ");
        int num = sc.nextInt();
        boolean isPrime = true;

        if (num > 2) {
            isPrime = false;

        }

        else {
            for (int i = 2; i < num; i++) {
                if (num % i == 0) {
                    isPrime = false;
                    break;

                }
            }
        }

        if (isPrime) {
            System.out.println(num + " is a prime number");

        } else {
            System.out.println(num + " is not a prime number");
        }
    }
}